### ISSUE TITLE
capslock

### ISSUE THREAD

**Issue by basxto** [yuliskov/LeanKeyboard#62]
Other keyboards allow to enable capslock with a double click on ⇧ when something needs to be written all caps.

**Comment by yuliskov** [comment]
This feature should work exactly as you described. I'll check. Maybe it's a
bug.

Regards, Yuriy

**Comment by basxto** [comment]
I played a bit around with it and found capslock. With my gamepad I get capslock when I press the right analog stick and shift for the next character when I click the shift button on the keyboard.

When it’s a bug, I guess my hardware/software might be relevant:
* LeanKeyboard 6.1.23 downloaded from https://github.com/yuliskov/LeanKeyboard/releases/tag/6.1.23
  * German layout
* Fire TV Stick 4K Ultra HD running FireOS 6.7.0.8 (Android 7.1.2)
* Official Fire TV remote app
* Satechi ST-UBGC Bluetooth gamepad (ABXY, dpad, two analog sticks, four shoulder buttons, select, start and a function to use the right analog stick as a mouse)
(real remote is temporarily lost)

**Comment by yuliskov** [comment]
The caps should be activated by the double press on shift button. Have you
tried this shortcut?

Regards, Yuriy

**Comment by basxto** [comment]
Yes, it does work. I was just too slow, I expected a delay time that is 10x of what it actually is.
